src/main/java/com/dev/cinema/service/impl/MovieServiceImpl.java

```diff
@@ -16,16 +17,19 @@ public MovieServiceImpl(MovieDao movieDao) {
         this.movieDao = movieDao;
     }
 
+    @Transactional
```

Comment:
why have you decided to add this annotation to this class?

Reply:
its an experiment))